### DISCUSSION TITLE
PostgreSQL `format` works, but can't `mount` using Unix domain socket

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2604] [Q&A] [answered]
This works:
```
juicefs format \
    --storage s3 \
    --bucket https://*******.com \
    --access-key ******* \
    --secret-key ******* \
    "postgres:///postgretest?host=/run/postgresql&user=postgretest&password=postgretest" \
    postgretest
```

This dont:
```
export META_PASSWORD=postgretest
juicefs mount \
    --no-usage-report \
    --backup-meta 0 \
    --cache-size 0 \
    "postgres:///postgretest?host=/run/postgresql&user=postgretest" \
    /tmp/postgretest
```
Error is:
```
<FATAL>: invalid uri: postgres:///postgretest?host=/run/postgresql&user=postgretest
```
Params taken from https://juicefs.com/docs/community/databases_for_metadata/#postgresql

**Comment by zhijian-pro** [comment, marked as answer]
Its a bug, we will fix it.

**Reply by zhijian-pro** [reply]
Fixed by https://github.com/juicedata/juicefs/pull/2607

**Reply by solracsf** [reply]
@zhijian-pro I've tested the latest build `juicefs version 1.1.0-dev+2022-08-26.bb841274`, I've got the same error.
Has the syntax changed here? See my 1st post.

`<FATAL>: invalid uri: postgres:///postgretest?host=/run/postgresql&user=postgretest [interface.go:405]`

**Reply by solracsf** [reply]
Ok got it working after reading the PR for new syntax. Thanks 🆗
